math/find_min.cpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file
+ * @brief A simple program to check large value of a list.

```

Comment:
Function implements the minimum number in the list, but description says otherwise. 

Include a wiki link of the algorithm.

Reply:
Soorry for mistake

Reply:
Sorry for my mistake.